api/src/main/java/com/kodemon/api/dto/UserDTO.java

```diff
@@ -28,6 +28,7 @@
     private Set<BadgeDTO> badges = new HashSet<>();
     @JsonProperty(required = false, defaultValue = "[]")
     private List<PokemonDTO> pokemons = new ArrayList<>();
+    private List<PokemonDTO> activePokemons = new ArrayList<>();
```

Comment:
Add `@JsonIgnore` annotation here please. We do not need this to be sent via REST.